### DISCUSSION TITLE
Package to be used for connecting to Clarius Ultrasound systems

### DISCUSSION THREAD

**Discussion by daudrain** [PlusToolkit/PlusLib#843] [Q&A]
The [following page](https://plustoolkit.github.io/download) refers to a package for Clarius US probes but there is no existing link as escalated in issue #725.

Do you know have any link to this package or any steps to follow to generate it?

Thank you,
David

**Comment by Sunderlandkyl** [comment]
The Clarius package is not generated for the stable release, however you can find it in the nightly release: http://perk-software.cs.queensu.ca/plus/packages/nightly/.

**Reply by daudrain** [reply]
Thank you @Sunderlandkyl

**Reply by daudrain** [reply]
Nightly works fine with Clarius App 7.3.4 😄 , this is not the case with Clarius App 8.0.1 but that's fine for now.
Thank you

**Comment by ungi** [comment]
Right now, PLUS connects to the phone or tablet running Clarius software through the Listen API of Clarius. However, a new interface is also implemented in PLUS that connects to the OEM API. With OEM API, you can control and read Clarius machines without a phone/tablet, directly from your PC. It's essentially finished but not published yet. If you start building a new system with Clarius now, going the OEM API route may be the right path. If you are interested, I can connect you with the people who have a special PLUS built for the OEM API.

**Reply by daudrain** [reply]
Thank you @ungi, that's good to know as Clarius is updating a lot its APIs/libraries.

**Reply by genlinut** [reply]
Hello @ungi , thank you for this comment.

As I actually plan to buy a probe from Clarius for research purposes, i asked myself, if i need one or multiple of their additional  [packages](https://clarius.com/scanners/research/). For example the raw data package?

And it seems, that the actual OEM API is implemented -> [commit](https://github.com/PlusToolkit/PlusLib/commit/104c117885b2a6e68ac34fdab156531677f73923)

**Reply by ungi** [reply]
As far as I know, Cast API still requires a mobile device. OEM API support in PLUS is still not public.
